mws/apimws/fixtures/phplibs.yaml

```diff
@@ -1,6 +1,7 @@
 - model: apimws.PHPLib
   fields:
     name: libawl-php
+    name_next_os: libawl-php
```

Comment:
Did we decide that we'll never have ``name_next_next_os``? :)

Reply:
My reasoning was that when all are migrated to stretch and you are ready for the next upgrade you will be able to migrate name_next_os -> name, re-populate name_next_os, and update NEXT_OS.